programs/uxd/tests/integration_tests/api/program_credix/accounts/find_pdas.rs

```diff
@@ -32,12 +32,26 @@ pub fn find_liquidity_pool_token_account(
     spl_associated_token_account::get_associated_token_address(signing_authority, base_token_mint)
 }
 
-pub fn find_treasury(multisig: &Pubkey) -> Pubkey {
-    *multisig // The treasury is the same key as the multisig on mainnet
+pub fn find_credix_treasury(multisig: &Pubkey) -> Pubkey {
+    *multisig // The treasury is an abritrary wallet that collect the fees
 }
 
-pub fn find_treasury_pool_token_account(treasury: &Pubkey, base_token_mint: &Pubkey) -> Pubkey {
-    spl_associated_token_account::get_associated_token_address(treasury, base_token_mint)
+pub fn find_credix_treasury_token_account(
+    credix_treasury: &Pubkey,
+    base_token_mint: &Pubkey,
+) -> Pubkey {
+    spl_associated_token_account::get_associated_token_address(credix_treasury, base_token_mint)
+}
+
+pub fn find_treasury_pool(multisig: &Pubkey) -> Pubkey {
+    *multisig // The treasury_pool is an abritrary wallet that collect the fees
+}
```

Comment:
this function and `find_treasury_pool` and `find_credix_treasury` look similar.

Reply:
For sure, this is aimed at abstracting away how those wallets are generated in the first place, and those are two separate PDA that you mentionned have no relation to each other 👌